src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -855,3 +855,46 @@ Tests:
     - "int" : { "$numberLong" :"1"}
     - "int" : { "$numberLong" :"2"}
     - "int" : { "$numberLong" :"3"}
+
+  - Name: Cycle through three pre-computed elements
+    GivenTemplate:
```

Comment:
Please add some examples to https://github.com/mongodb/genny/blob/master/src/workloads/docs/Generators.yml also. That's what I was trying to come up with during standup. Those are actual docs.

Reply:
Added an example for both new generators to Generators.yml